test/extended/templates/templateservicebroker_bind.go

```diff
@@ -59,7 +58,7 @@ var _ = g.Describe("[Conformance][templates] templateservicebroker bind test", f
 				},
 				Subjects: []corev1.ObjectReference{
 					{
-						Kind: authorizationapi.GroupKind,
```

Comment:
promote this constant to openshift/api

Reply:
https://github.com/openshift/api/pull/351 will pick together with library-go

Reply:
picked, fixed.